### ISSUE TITLE
Rendering .md Files with Quotation Marks

### ISSUE THREAD

**Issue by versipellis** [GenieFramework/Genie.jl#129]
**Describe the bug**
Any `.md` files with quotation marks will throw an error 500 when using `!html` to render:
`LoadError: syntax: cannot juxtapose string literal`

**To Reproduce**
Use `html!` in the controller to render a `.md` file with quotation marks: `"`

**Expected behavior**
Renders markdown fine.

**Additional context**
`julia> versioninfo()`
```
julia> versioninfo()
Julia Version 1.1.0
Commit 80516ca202 (2019-01-21 21:24 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
Environment:
  JULIA_EDITOR = atom  -a
  JULIA_NUM_THREADS = 4
  JULIA_REVISE = auto
```

`pkg> st`
```
(edatool) pkg> st
    Status `~/Projects/juliaeda/edatool/Project.toml`
  [c43c736e] Genie v0.8.9 #master (https://github.com/essenciary/Genie.jl)
  [682c06a0] JSON v0.20.0
  [39ec1447] Millboard v0.2.0
  [295af30f] Revise v1.1.0
  [ddb6d928] YAML v0.3.2
```

**Comment by essenciary** [comment]
Thanks, I think I can patch it quickly

**Comment by essenciary** [comment]
@versipellis Sorry, but this is more difficult than I thought... For now the recommended way is to escape quotes in the markdown view... I'll try to find a solution.

**Comment by versipellis** [comment]
Not a problem! Not a high-priority issue for me, just something I thought I'd point out.

**Comment by essenciary** [comment]
Had another look at this - tried a few things but still no dice. The only working approach so far is to escape the quotes `\"...\"`.

**Comment by essenciary** [comment]
@versipellis Solved on #master and upcoming v0.11.0 💪
